vast.conf.example

```diff
@@ -20,9 +20,9 @@ system {
   ;node-id = "node"
 
   ; List of paths to look for schema files in ascending order of priority.
-  ; Note: Automatically prepended with
-  ;  ["<binary_directory>/../share/vast/schema", "/etc/vast/schema"].
-  ; Use the no-default-schema option to turn off this mechanism.
+  ; Note: Automatically prepended with "/etc/vast/schema" on Linux systems, and 
+  ; ["<binary_directory>/../share/vast/schema", "/etc/vast/schema"] on other
+  ; platforms. Use the no-default-schema option to turn off this mechanism.
```

Comment:
I don't understand why you changed the schema loading paths as well, I thought this was only about `vast.conf`?

Reply:
So the handling of the `share/vast` directory is consistent for all data kept in it on any given platform.